llama_index/readers/chroma.py

```diff
@@ -39,7 +41,8 @@ def __init__(
 
         self._client = chromadb.Client(
             Settings(
-                chroma_api_impl="rest",
+                chroma_api_impl=chroma_api_impl,
+                chroma_db_impl=chroma_db_impl or "chromadb.db.duckdb.DuckDB",
```

Comment:
is "chromadb.db.duckdb.DuckDB" the default string value?

Reply:
I believe so
https://github.com/chroma-core/chroma/blob/main/chromadb/config.py#L42